src/app/pages/questionnaire-page/questionnaire-page.component.html

```diff
@@ -19,12 +19,12 @@ <h2 matDialogTitle style="font-size: 2rem">{{'Food Frequency Questionnaire' | tr
         <mat-form-field>
           <input matInput value="infantage" class="qb-row-item" [(ngModel)]=infantage type="number" min="1" max="24" step="1" pattern="\d+" placeholder=" {{'In months' | translate}} " autocomplete="off" required>
         </mat-form-field>
-        <mat-error *ngIf="!infantage">
+        <mat-error *ngIf="decimalError">
```

Comment:
This is the first confusion that whoever did this first put in place. They were not supposed to do any calculations like !infantage or infantage % 1 != 0 in the html tab. Just calling up the 'decimalError' or 'genderError' which already should be in the .ts file for other portals